integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/DatadogInterceptor.kt

```diff
@@ -397,6 +417,48 @@ internal constructor(
 
     // endregion
 
+    // region Builder
+
+    /**
+     * A Builder for the [DatadogInterceptor].
+     */
+    open class Builder(tracedHosts: Map<String, Set<TracingHeaderType>>) :
```

Comment:
We probably need to remove `open` modifier from here and `DatadogInterceptor` as well. I think it was originally added because we have `RumInterceptor : DatadogInterceptor`.

But removing it involves a risk - some customers may have already sub-classed `DatadogInterceptor`.

Reply:
I don't want to do that, at least not now.